root/layout/components/ExternalLinks.js

```diff
@@ -87,10 +87,18 @@ const ExternalLinks = ({entity, empty, heading}) => {
   }
 
   otherLinks.sort(function (a, b) {
-    return compare(a.target.sidebar_name, b.target.sidebar_name);
+    return compare(a.target.sidebar_name, b.target.sidebar_name) ||
+      compare(a.target.href_url, b.target.href_url);
   });
 
-  links.push.apply(links, otherLinks.map(function (relationship) {
+  const uniqueOtherLinks = otherLinks.filter(
+    function (link, index, sortedLinks) {
+      return index === 0 ||
+        sortedLinks[index - 1].target.href_url !== link.target.href_url;
```

Comment:
Would `const uniqueOtherLinks = _.uniqBy(otherLinks, x => x. target.href_url)` (using `uniqBy` from lodash) also do the trick? I think that's what the sort-and-compare-pairwise logic does, but hard to be sure :)

Reply:
Ah true. Maybe it can be sorted after the uniq instead of before, but probably not worth futzing with.

Reply:
I just found that Lodash provides [`sortedUniqBy`](https://lodash.com/docs#sortedUniqBy) too! I used it with your first suggestion in https://github.com/metabrainz/musicbrainz-server/pull/641/commits/4ad24ac319205b1df009fc62ad721922b961195.